pbconf/python-redfish.pb

```diff
@@ -130,7 +130,7 @@ defpkgdir python-redfish = .
 
 # List of files per pkg on which to apply filters
 # Files are mentioned relatively to pbroot/defpkgdir
-filteredfiles python-redfish = redfish-client/redfish-client,doc/source/conf.py
+filteredfiles python-redfish = redfish-client/redfish-client,doc/source/conf.py,setup.py,./redfish-client/etc/redfish-client.conf,install.sh
```

Comment:
Ok. But it's not setup.py role to replace PB variables either :-( That"s pb's role. setup.py should completely ignore that mechanism.